EIPS/eip-20.md

```diff
@@ -119,21 +104,16 @@ The function SHOULD `throw` unless the `_from` account has deliberately authoriz
 function transferFrom(address _from, address _to, uint256 _value) public returns (bool success)
 ```
 
-
-
 #### approve
 
 Allows `_spender` to withdraw from your account multiple times, up to the `_value` amount. If this function is called again it overwrites the current allowance with `_value`.
 
-**NOTE**: To prevent attack vectors like the one [described here](https://docs.google.com/document/d/1YLPtQxZu1UAvO9cZ1O2RPXBbT0mooh4DYKjA_jp-RLM/) and discussed [here](https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729),
-clients SHOULD make sure to create user interfaces in such a way that they set the allowance first to `0` before setting it to another value for the same spender.
-THOUGH The contract itself shouldn't enforce it, to allow backwards compatibility with contracts deployed before
+**NOTE**: Please read the [Security Considerations](#security-considerations) section for potential attack vectors.
```

Comment:
Seems like you're removing part of the specification?

```suggestion
clients SHOULD make sure to create user interfaces in such a way that they set the allowance first to `0` before setting it to another value for the same spender.
THOUGH The contract itself shouldn't enforce it, to allow backwards compatibility with contracts deployed before

**NOTE**: Please read the [Security Considerations](#security-considerations) section for potential attack vectors.
```

Reply:
I didn't think so. I'll readd it if it means this can be merged.